backend/api.py

```diff
@@ -311,4 +316,83 @@ async def websocket_pcap(websocket: WebSocket):
                 continue
 
     except WebSocketDisconnect:
-        print("Client disconnected from /ws/gnb")
\ No newline at end of file
+        print("Client disconnected from /ws/gnb")
+
+@app.get("/gnb/config")
+def show_gnb_config():
+    """
+    Return a list of all .yaml or .conf files that contain 'gnb' in their name
+    from the predefined configuration directory.
+
+    Returns:
+        dict: A dictionary with a list of matching file names.
+    """
+    try:
+        if not CONFIG_DIR.exists():
+            raise HTTPException(status_code=404, detail="Configuration directory not found.")
+
+        files = [
+            f.name for f in CONFIG_DIR.iterdir()
+            if f.is_file() and "gnb" in f.name.lower() and f.suffix in [".yaml", ".conf"]
+        ]
+
+        return {"files": files}
+
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"Failed to list config files: {str(e)}")
+
+@app.post("/metrics/push")
+def push_metrics():
```

Comment:
Nope, The current file is responsable for implement the API. So this file have to send request to influxDB not to send data. 

This should be in another file named like `influxDB_example_command` if you would like to create a file responsible for sending fake data to the gNB.